test.py

```diff
@@ -0,0 +1,65 @@
+import os
+import pytest
+import subprocess
+import testinfra
+
+@pytest.fixture(scope='session')
+def host(request):
+    subprocess.check_call(['docker', 'build', '-t', 'radicale', '.'])
+    docker_id = subprocess.check_output(['docker', 'run', '-d', 'radicale']).decode().strip()
+    
+    yield testinfra.get_host("docker://" + docker_id)
+    
+    # teardown
+    subprocess.check_call(['docker', 'rm', '-f', docker_id])
+
+def test_system(host):
+    assert host.system_info.distribution == 'alpine'
+    assert host.system_info.release == '3.7.0'
+
+def test_entrypoint(host):
+    entrypoint = '/usr/local/bin/docker-entrypoint.sh'
+    assert host.file(entrypoint).exists
+    assert oct(host.file(entrypoint).mode) == '0o555'
+
+def test_process(host):
+    process = host.process.get(comm='radicale')
+    assert process.pid == 1
+    assert process.user == 'radicale'
+    assert '/usr/bin/radicale --config /config/config' in process.args
+
+def test_port(host):
+    assert host.socket("tcp://0.0.0.0:5232").is_listening
+
+def test_radicale_version(host):
+    assert host.check_output("/usr/bin/pip3 --disable-pip-version-check show radicale | grep Version | awk -F ' ' '{print $2}' | tr -d '\n'") == os.environ.get('VERSION','2.1.9')
+
+def test_radicale_user(host):
```

Comment:
would skip this test

Reply:
Is it worth leaving in until we have black box tests in place? I've had a quick look but can't see a "version" or "status" like endpoint to curl against a running radicale